src/org/thoughtcrime/securesms/WebxdcActivity.java

```diff
@@ -101,6 +101,7 @@ private static Intent[] getWebxdcIntentWithParentStack(Context context, int msgI
   @Override
   protected void onCreate(Bundle state, boolean ready) {
     super.onCreate(state, ready);
+    setForceDark();
```

Comment:
this was just missing - @adbenitez or do you know of a reason that it was not there?